app/javascript/components/vm-server-relationship-form/index.jsx

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Grid } from 'patternfly-react';
+import MiqFormRenderer from '../../forms/data-driven-form';
+import { API } from '../../http_api';
+import { cleanVirtualDom } from '../../miq-component/helpers';
+import createSchema from './vm-server-relationship-form.schema';
+
+class VmServerRelationShipForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      initialValues: {
+        serverId: null,
+      },
+      schema: createSchema([]),
```

Comment:
you don't need this because you are not rendering it before finishing initial data request